content/git/concepts/pull-requests/pull-requests.md

```diff
@@ -26,15 +26,42 @@ PRs are featured on popular version control platforms such as [GitHub](https://g
 
 ## Create a Pull Request
 
-The prerequisites for creating a PR and contributing to a project are:
+### Method 1: Fork (Ideal for Open Source Projects)
 
 1. Fork the project.
-2. Clone the forked repository to local machine.
-3. Create, and switch into, a new branch.
-4. Add, commit, and push the changes back to the repository.
+2. Clone the forked repository to your local machine.
+3. Create a new branch, then switch to it.
+4. `add`, `commit`, and `push` the changes to your forked repository.
+5. From the web page of **YOUR** forked repository, click on the "Pull requests" tab.
 
-With the newly added changes pushed, open the web page of the repository. There should be a button that says "Create Pull Request" placed somewhere, similar to the image below.
+### Method 2: Collaborators (Ideal for Small Teams)
+If you are a collaborator of a repository (meaning the repository owner has added you as a collaborator), you can:
 
-![Repo page with Pull Request detect button](https://i.imgur.com/TsKCnyr.png)
+1. Clone the repository.
+2. Create a new branch, then switch to it.
+3. Make your changes, `commit`, and then `push` the branch to the repository.
+4. From the web page of the repository, click on "Pull Requests" tab.
 
-Click that button and follow the steps to create the PR.
+### Method 3: Within an Organization/Company
+If you are part of an organization/company, depending on your permissions, you may have the privilege to push branches directly and create PRs (Pull Requests).
+
+1. Clone the repository.
+2. Create and switch to a new branch.
+3. Make your changes, `commit`, and then `push` the branch to the repository.
+4. From the web page of the repository, click on "Pull Requests".
+
+> [!IMPORTANT]
+> Regardless of the method you choose, be sure to follow the contribution guidelines and
+> standards specific to the project you are working on.
```

Comment:
```suggestion
> **Note:** Regardless of the method you choose, be sure to follow the contribution guidelines and standards specific to the project you are working on.
```

Reply:
In the doc I followed on GitHub concerning Markdown it is specified that the `>**CALLOUT**` method is obsolete or at least will become so

[Link to documentation](https://docs.github.com/fr/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

Should I make the change anyway?

Reply:
In terms of our documentation, we follow these [guidelines](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md). Please go through it and make the changes.